dev/com.ibm.ws.config/resources/com/ibm/ws/config/internal/resources/ConfigMessages.nlsprops

```diff
@@ -170,10 +170,13 @@ error.unique.value.conflict=CWWKG0031E: The value {1} specified for unique attri
 error.unique.value.conflict.explanation=The value for the specified attribute must be unique but it is already being used.
 error.unique.value.conflict.useraction=Change the attribute value so that it is unique.
 
-warn.config.invalid.value=CWWKG0032W: Unexpected value specified for property [{0}], value = [{1}]. Expected value(s) are: {2}.
+# {3} is either the formatted value of default.value.in.use or an empty string
+warn.config.invalid.value=CWWKG0032W: Unexpected value specified for property [{0}], value = [{1}]. Expected value(s) are: {2}. {3} 
```

Comment:
**change**
Unexpected value specified for property [{0}]
**to**
Unexpected value specified for the [{0}] property

Reply:
This message has been in Liberty for years. Given our guidelines against changing messages in service, this should probably stay the way it is.

Reply:
ok- understood. In that case, is the "ID review not needed" tag appropriate?